pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConsumerConfigurationData.java

```diff
@@ -114,7 +114,7 @@ public int getMaxPendingChuckedMessage() {
 
     private boolean autoAckOldestChunkedMessageOnQueueFull = false;
 
-    private long expireTimeOfIncompleteChunkedMessageMillis = 60 * 1000;
+    private long expireTimeOfIncompleteChunkedMessageMillis = TimeUnit.HOURS.toMillis(1);
```

Comment:
I think we should update the doc. 1 hour is too long for most cases. @merlimat @rdhabalia Please help confirm.

Reply:
+1 .

Reply:
+1 for updating the doc. If there are no other objections, I will update the doc.